ui/src/containers/NodePage.js

```diff
@@ -32,7 +41,10 @@ const NodePage = (props) => {
           ]}
         />
       </BreadcrumbContainer>
-      <NodePageContent nodes={nodes}></NodePageContent>
+      <NodePageContent
+        nodes={nodes}
+        nodeListData={nodeListData}
```

Comment:
Right. It is very confusing. so maybe I should rename `nodeListData` to `nodeTableData` which is used by the NodeListTable component.

Reply:
What difference? Can't we make sure we have all we need in a single array?

Reply:
See for me, the `nodeTableData` should only include the data needed for the table, not more than that.
And the `nodes` is the object return by k8s with more information. 

Here actually we don't need to fetch the `nodes`. Since it's only needed in the health tabs. We should fetch it in the lower child component `<NodeRSP>`